frontend/src/components/PreferencesCheckbox.js

```diff
@@ -23,7 +23,7 @@ export default function PreferencesCheckbox(props) {
           onChange={(event) => props.onChange(event.target.checked)}
           name="checkbox"
           color="primary"
-          defaultChecked="true"
+          defaultChecked={true}
```

Comment:
Just curious: Any particular reason to make this change? To fix a bug, or to format?

Reply:
This change was to avoid any type mismatch as the default value would not be the string value but the boolean true.

Reply:
thanks for the explanation